### DISCUSSION TITLE
How do I get phone number of a user from Google and Facebook?

### DISCUSSION THREAD

**Discussion by qnxdev** [nextauthjs/next-auth#2665] [Help]
### Question 💬

My specific application requires users to submit phone number for OTP verification. 
How do I get phone number from Google and Facebook after user signs in, while using Next-Auth ? 

I use Google and Facebook login for other personal details that can automatically be used to fill the user profile in my site.

Thanks in advance

### How to reproduce ☕️

Create next app

Add next-auth

Get phone number of a signed in user or while the user signs in first time

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by balazsorban44** [comment]
You will have to add extra scopes to your provider config. Unfortunately I don't know the name of the scopes for phone number in these providers, so you have to research a bit

**Reply by qnxdev** [reply]
I got the scope

It is called `https://www.googleapis.com/auth/admin.directory.user.readonly`

It is a Google Workspace scope and I think it'll be only working for some Enterprise setup which usues Workspace.

Sad
